src/views/Layout.jsx

```diff
@@ -39,10 +39,7 @@ export function Layout() {
 				</main>
 				<footer className="Layout-footer">
 					<p className="Layout-footer-attribution">
-						Crafted with care{' '}
-						<span role="img" aria-label="purple-heart">
-							💜{' '}
-						</span>
+						Crafted with care <Emoji label="purple-heart">💜</Emoji>
 						by{' '}
```

Comment:
Super nit: suggesting a space after heart emoji
```suggestion
						{' '}by{' '}
```